configure

```diff
@@ -146,7 +146,7 @@ if [ "$OPT" = 'yes' ]; then
 fi
 
 if [ "$X11" = 'yes' ]; then
-	echo 'xlib = -L/usr/X11/lib -lX11' >>Makefile
+	echo 'xlib = -L/usr/X11/lib -lX11 -lXtst -lm' >>Makefile
```

Comment:
Maybe we should test for the existence of the Xtst extension library and make this feature a build-time configuration option. But I don't consider this a blocking issue. This can be implemented after the merge.